typescript/unit_test.go

```diff
@@ -0,0 +1,27 @@
+package typescript
```

Comment:
I get the naming convention, but this is the worst name for a file in the history of ever

Reply:
We can think of renaming "Unit" into something else. It meant to be unit of compilation. I guess "File" might work as well. I think I borrowed this "Unit" term from Roslyn. The only thing it will be renamed everywhere: C#, Scala...

Reply:
I would think thats a bit better, but it's not really necessary